### PR TITLE
Aa size fix

### PR DESCRIPTION
**Pull request by dlangston**

### CODE REVIEW

**Comment by dlangston** [comment]
before.... 
screen shot 2015-08-28 at 12 49 40 pm

after....
screen shot 2015-08-28 at 12 49 29 pm

File: ooiui/static/css/common/plotting.css [hunk 1]
```diff
@@ -134,6 +134,10 @@
     background: transparent;
   }
 
+  div#highcharts-0 {
+    width: auto!important;
```
Comment (birdage) [inline ooiui/static/css/common/plotting.css]: @dlangston why the important, we should try and reframe from using this

File: ooiui/templates/aa/AlertPage.html [hunk 2]
```diff
@@ -49,7 +49,7 @@
                 </button>
               </i>
             </div>
-            <div class="row form-group">
+            <div class="row form-group" style="width:100%; padding-left:30px;">
```
Comment (birdage) [inline ooiui/templates/aa/AlertPage.html]: @dlangston this should be in a css class

File: ooiui/templates/science/plotting.html [hunk 3]
```diff
@@ -56,7 +56,7 @@
                 </div>
                     <div class="row">
                         <div class="panel">
-                            <div class="panel-heading">
+                            <div class="panel-heading" style="padding-left:0px;">
```
Comment (birdage) [inline ooiui/templates/science/plotting.html]: should be in a css class

**Comment by dlangston** [comment]
@birdage

**Comment by birdage** [comment]
@dlangston i cant merge, there is a merge conflict

**Comment by dlangston** [comment]
ok I will take a look

**Comment by dlangston** [comment]
@birdage fixed the conflict but i realized my fix didn't work for a browser resolution at 1024. Plan to add a scrollable table.
screen shot 2015-09-16 at 10 03 06 am

**Comment by DanielJMaher** [comment]
@dlangston Merge Conlfict.  Fix.

**Comment by dlangston** [comment]
@DanielJMaher I have fixed the merge conflict in a separate branch that i haven't pushed yet because I still need to add the scrollable table

**Comment by birdage** [comment]
@dlangston can we close this then?

**Pull request closed without merging**